cmd/importverifier/importverifier.go

```diff
@@ -276,3 +291,47 @@ func logForbiddenPackages(base string, forbidden []string) {
 		log.Printf("--- %s\n", forbiddenPackage)
 	}
 }
+
+// Check if any imports are disallowed
+// Exec e.g find . -name "*.go" |  grep -v vendor/ | grep -v cmd/kubeadm/app | { xargs grep 'github.com/pkg/errors' -sl || true; }
+func verifyDisallowedImports() error {
+
+	// Manually create a single entry for github/pkg/errors
+	disallowedImports := []DisallowedImport{{
+		Pkg:          "github.com/pkg/errors",
+		IgnoredTrees: []string{"cmd/kubeadm/app", "cmd/importverifier"},
```

Comment:
i think ideally, instead of hardcoding paths and packages in the tool source code this should be on the side of the wrapping bash scripts - i.e. passed as flags.

Reply:
good point, thanks

Reply:
Well, I think creating a slice of `DisallowedImport` (Probably a config yaml file ) and passed to the Go script ?

Reply:
probably better to have it as flags instead of config for the time being, since i don't think the tool supports other config files yet, IIRC.